tests/utils_tests/test_datetime_safe.py

```diff
@@ -3,10 +3,14 @@
     time as original_time,
 )
 
-from django.test import SimpleTestCase
-from django.utils.datetime_safe import date, datetime, time
+from django.test import SimpleTestCase, ignore_warnings
+from django.utils.deprecation import RemovedInDjango50Warning
 
+with ignore_warnings(category=RemovedInDjango50Warning):
+    from django.utils.datetime_safe import date, datetime, time
 
+
+# RemovedInDjango50Warning
```

Comment:
this also.

Reply:
Ditto. (See [here](https://github.com/django/django/commit/028f10fac651292edf8d50e013b16a52862e4c08#diff-d748a78a482295bf93556048d13acb2b8187f8dc0462cc04ff12109774712cb2R12).)